orders/src/main/java/com/orders/crudyorders/repos/OrdersRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.orders.crudyorders.repos;
+
+import com.orders.crudyorders.model.Customer;
+import com.orders.crudyorders.model.Order;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.List;
+
+public interface OrdersRepository extends CrudRepository<Order, Long>{
+    List<Order> findAllByCustomer(Customer customer);
+
+}
```

Comment:
Line 10 - the naming for this method is actually incorrect - see code comment above (on your `CustomersRepository.java` file).